node/network/bridge/src/validator_discovery.rs

```diff
@@ -82,39 +120,19 @@ impl<N: Network, AD: AuthorityDiscovery> Service<N, AD> {
 			}
 		}
 
-		let state = &mut self.state[peer_set];
-		// clean up revoked requests
-		let multiaddr_to_remove: HashSet<_> =
-			state.previously_requested.difference(&newly_requested).cloned().collect();
-		let multiaddr_to_add: HashSet<_> =
-			newly_requested.difference(&state.previously_requested).cloned().collect();
-		state.previously_requested = newly_requested;
-
 		tracing::debug!(
 			target: LOG_TARGET,
 			?peer_set,
 			?requested,
-			added = multiaddr_to_add.len(),
-			removed = multiaddr_to_remove.len(),
 			?failed_to_resolve,
 			"New ConnectToValidators request",
 		);
```

Comment:
this debug log is printed twice now

Reply:
Where is the second time?

Reply:
in `on_resolved_request` but with a bit more details

Reply:
Uh ok, that was on purpose to see the code path in the logs. Probably not needed - will remove it.

Reply:
or not - it also adds info about resolve errors, that was the reason why I kept it.